module-apps/application-settings/windows/display-keypad/QuotesAddWindow.cpp

```diff
@@ -158,11 +158,10 @@ namespace gui
                 quoteModel->edit(quoteData);
             }
 
-            auto backToOptionWindow = 1;
-            auto backToMainWindow   = 2;
-
-            auto windowToBack = quoteAction == QuoteAction::Add ? backToOptionWindow : backToMainWindow;
-            application->returnToPreviousWindow(windowToBack);
+            if (quoteAction != QuoteAction::Add) {
+                application->switchWindow(gui::name::window::main_window);
```

Comment:
hmm that may not be the same ? or we just ignore it as Quotes are not finished?

Reply:
I was trusting the developer who wrote "backToManiWindow" I'll have to double-check with QA :) (if the code is alive at least)